CRM/Volunteer/Hook.php

```diff
@@ -0,0 +1,13 @@
+<?php
+class CRM_Volunteer_Hook
+{
+  static $_nullObject = NULL;
```

Comment:
Don't think you need this; I think you can use CRM_Utils_Hook::$_nullObject instead.